roles/splunk_cluster_master/tasks/main.yml

```diff
@@ -11,6 +11,14 @@
   when:
     - splunk.apps_location
 
+- include_tasks: ../../splunk_common/tasks/set_as_hec_receiver.yml
+  vars:
+    splunk_hec_inputs_app_path: "apps/splunk_httpinput/local"
```

Comment:
We should already have a `splunk.app_paths.httpinput` path for this (minus the `local`)